react/package.json

```diff
@@ -6,15 +6,15 @@
   "main": "./dist/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "yarn && babel src --presets babel-preset-env --out-dir dist",
+    "build": "yarn && tsc",
     "link:themes": "bash ./linkThemes.sh",
     "start": "cd demo && yarn && cd .. && yarn link:themes && cd demo && yarn start"
   },
   "dependencies": {
+    "@material-ui/core": "^4.0.0",
```

Comment:
Our themes package should not depend on the components library...can we import just the type definitions from somewhere in @types/XXXXXXX instead?

Reply:
You're right this should have been the @types library.  I've made the changes and added @types/material-ui as a dependency now instead of @material-ui/core.